packages/accounts/src/utils.ts

```diff
@@ -0,0 +1,105 @@
+ * @see [https://docs.near.org/api/rpc/contracts#view-contract-state](https://docs.near.org/api/rpc/contracts#view-contract-state)
+ *
+ * @param connection connection to query state from
+ * @param accountId account whose state is viewed
+ * @param prefix allows to filter which keys should be returned. Empty prefix means all keys. String prefix is utf-8 encoded.
+ * @param blockQuery specifies which block to query state at. By default returns last "optimistic" block (i.e. not necessarily finalized).
+ */
+export async function viewState(connection: Connection, accountId: string, prefix: string | Uint8Array, blockQuery: BlockReference = { finality: 'optimistic' }): Promise<Array<{ key: Buffer; value: Buffer }>> {
+    const { values } = await connection.provider.query<ViewStateResult>({
+        request_type: 'view_state',
+        ...blockQuery,
+        account_id: accountId,
+        prefix_base64: Buffer.from(prefix).toString('base64')
+    });
+
+    return values.map(({ key, value }) => ({
+        key: Buffer.from(key, 'base64'),
+        value: Buffer.from(value, 'base64')
+    }));
+}
+
+
+/**
+ * Invoke a contract view function using the RPC API.
+ * @see [https://docs.near.org/api/rpc/contracts#call-a-contract-function](https://docs.near.org/api/rpc/contracts#call-a-contract-function)
+ *
+ * @param options Function call options.
+ * @param options.contractId NEAR account where the contract is deployed
+ * @param options.methodName The view-only method (no state mutations) name on the contract as it is written in the contract code
+ * @param options.args Any arguments to the view contract method, wrapped in JSON
+ * @param options.parse Parse the result of the call. Receives a Buffer (bytes array) and converts it to any object. By default result will be treated as json.
+ * @param options.stringify Convert input arguments into a bytes array. By default the input is treated as a JSON.
+ * @param options.jsContract Is contract from JS SDK, automatically encodes args from JS SDK to binary.
+ * @param options.blockQuery specifies which block to query state at. By default returns last "optimistic" block (i.e. not necessarily finalized).
+ * @returns {Promise<any>}
+ */
+export async function viewFunction(connection: Connection, {
+    contractId,
+    methodName,
+    args = {},
+    parse = parseJsonFromRawResponse,
+    stringify = bytesJsonStringify,
+    jsContract = false,
+    blockQuery = { finality: 'optimistic' }
+}: ViewFunctionCallOptions): Promise<any> {
+    let encodedArgs;
+
+    validateArgs(args);
+
+    if (jsContract) {
+        encodedArgs = encodeJSContractArgs(contractId, methodName, Object.keys(args).length > 0 ? JSON.stringify(args) : '');
+    } else {
+        encodedArgs = stringify(args);
+    }
+
+    const result = await connection.provider.query<CodeResult>({
+        request_type: 'call_function',
+        ...blockQuery,
+        account_id: jsContract ? connection.jsvmAccountId : contractId,
```

Comment:
Everything connected to `jsvmAccountId` is obsolete and will be removed very soon from near-api-js. Let's not add it to new functionality.

Reply:
The PR solves another issue and doesn't know anything about `jsvmAccountId`, this particular change has taken away function from the `Account` class and moved into another place
I wouldn't remove `jsvmAccountId` right here as it's a **breaking change** that may cause issues if you don't mind (and I assume you're preparing another PR out there that does remove it)